### ISSUE TITLE
Add optional campaign overlay showing unlocks under each planet

### ISSUE THREAD

**Issue by completesomething** [ZeroK-RTS/Chobby#868]
This would help in cases where you'd like hints on finding a specific planet/region for specific missions. For example, with the suggested overlay, it should be much easier to find the Kodachi Rally mission or the Terraforming mission.

**Comment by sprunk** [comment]
Perhaps just enable the debug view that draws unlocks at planets by default? For noticeability, could have a fairly prominent button/checkbox right on the galaxy map (instead of in options) to shows extra info.

**Comment by completesomething** [comment]
Yes, I was thinking that it would be a checkbox on the galaxy map, not in the lobby menu settings.
